pkg/operation/botanist/component/etcd/bootstrap_test.go

```diff
@@ -420,7 +429,6 @@ status: {}
 				gomock.InOrder(
 					c.EXPECT().Get(ctx, kutil.Key(namespace, managedResourceSecretName), gomock.AssignableToTypeOf(&corev1.Secret{})),
 					c.EXPECT().Update(ctx, gomock.AssignableToTypeOf(&corev1.Secret{})).Do(func(ctx context.Context, obj client.Object, opts ...client.UpdateOption) {
-						Expect(obj).To(DeepEqual(managedResourceSecret))
```

Comment:
why is this line removed?

Reply:
I am also not sure why this check is removed. Isn't it needed?

Reply:
Sorry, oversight. I added back in the updated PR.